src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,39 @@
+// Variables
+$color-moyo-blue: #00acdc;
+$color-main-accent: #060b35;
+$color-white: #fff;
+$color-secondary: #616070;
+$color-darkblue: #34568b;
+
+$box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+$dim-main-padding: 10px;
+$dim-main-margin: 50px;
+$dim-fontsize-big: 2rem;
+$dim-fontsize-small: 0.75rem;
+$dim-fontsize-detail: 10px;
+$dim-logo-size: 40px;
+$dim-menu-underscore: 4px;
+$dim-nav-gap: 1.25rem;
+$dim-container-margin: 50px;
+$dim-grid-gaps: 46px 48px;
+$dim-card-width: 200px;
+$dim-card-border-radius: 5px;
+$dim-star-size: 1rem;
+
+// Placeholders
+%flex-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+%flex-align-stretch {
+  display: flex;
+  align-items: stretch;
+}
+
+%pointer {
+  cursor: pointer;
+  user-select: none;
```

Comment:
You don't want to disable `user-select` on clickable elements. Some users may need that feature, always think about accessibility.